src/health/health.controller.ts

```diff
@@ -24,7 +24,7 @@ export class HealthController {
       () =>
         this.http.pingCheck(
           'ethconnect',
-          `${this.tokensService.baseUrl}/status`,
+          `${this.tokensService.baseUrl}/api`,
```

Comment:
We need to makes this backwards compatible though

Reply:
Will add an env var once I'm done testing the patch

Reply:
Actually this should also be compatible with Ethconnect... so maybe this is the right endpoint to use. Or we should add a `/status` to EVMConnect? Would want @peterbroadhurst to weigh in here.